docs/cookbook/extending-product-list.md

```diff
@@ -0,0 +1,129 @@
+# Extending Product List
```

Comment:
the cookbook is not linked anywhere... Please add it to cookbooks and maybe to article about extending entity